run/events-pubsub/app.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START run_events_pubsub_server_setup]
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+app.use(bodyParser.json());
+// [END run_events_pubsub_server_setup]
+
+// [START run_events_pubsub_handler]
+app.post('/', (req, res) => {
+  if (!req.body) {
+    const msg = 'no Pub/Sub message received';
+    res.status(400).send(`Bad Request: ${msg}`);
+    return;
+  }
+  if (!req.body.message) {
+    const msg = 'invalid Pub/Sub message format';
+    res.status(400).send(`Bad Request: ${msg}`);
+    return;
+  }
+  const pubSubMessage = req.body.message;
+  const name = pubSubMessage.data
+    ? Buffer.from(pubSubMessage.data, 'base64').toString().trim()
+    : 'World';
+  
+  res.send({
```

Comment:
You aren't logging as specified in your design doc

Reply:
This HTTP response looks good, but yes, more logging expected.

Reply:
Yup, I formalized the responses in the design doc a bit. Need to update it here.